powershell/Maester.psd1

```diff
@@ -12,7 +12,7 @@
 RootModule = 'Maester.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.2.0'
```

Comment:
You can leave this as-is. The build and deployment process automatically updates the module version when a normal release is published. This copy of the manifest stays unchanged during each of the preview releases that get frequently published.